pages/release-process.md

```diff
@@ -1,111 +1,113 @@
-
-When consensus has been reached on a release candidate, follow the steps below to complete the release using the chosen RC:
-
- 1. Merge your RC branch into master and push those commits upstream.  Afterwards, you can delete your RC branch.
-
-    ```shell
-    git checkout master
-    git merge 1.0.0-beta-RC
-    git push upstream master
-    git branch -d 1.0-0-beta-RC
-    ```
-
- 2. [Release the artifacts] in Sonatype OSS so that they get published in Maven Central.  You can drop any staging repos for RCs that were not chosen. 
-
- 3. Create a signed tag for the release from the chosen RC tag and push to upstream repo:
-
-    ```shell
-    # Create signed tag from RC2 tag.
-    # You may need to use -u <key-id> to specify GPG key
-    git tag -s 1.0.0-beta-1 1.0.0-beta-1-RC2
-    # Push signed tag to upstream repo
-    git push upstream 1.0.0-beta-1
-    ```
-
- 4. Attach Fluo tarball to GitHub release page.
-
- 5. Remove all RC tags
-
-    ```shell
-    # Remove tag locally
-    git tag -d 1.0.0-beta-1-RC1
-    # Remove tag on fork
-    git push --delete origin 1.0.0-beta-1-RC1
-    ```
- 6.  View the [website README] for instructions on how to generate Javadocs and documentation using the released tag.  Submit PR to the website repo to publish.
- 
- 7.  Send an email to `dev@fluo.incubator.apache.org` announcing new release.
-
-[website README]: https://github.com/apache/incubator-fluo-website/blob/master/README.md
-[documentation]: http://central.sonatype.org/pages/apache-maven.html
-[sonatype account]: https://issues.sonatype.org/
-[Release the artifacts]: http://central.sonatype.org/pages/releasing-the-deployment.html
+---
+layout: page
+title: Release Process
+permalink: /release-process/
+---
+
+### Initial Setup
+
+Before you can release Fluo or Fluo Recipes, you will need a GPG key. For information on generating
+a key look at this [ASF GPG page](https://www.apache.org/dev/openpgp.html).  After generating a key,
+add it to the [KEYS](https://www.apache.org/dist/incubator/fluo/KEYS) file.
+
+The maven release plugin will need credentials to stage artifacts.  You can provide the credentials
+by adding the following to your `~/.m2/settings.xml` file.  You may want remove your password after
+running maven.
+
+```xml
+<servers>
+  <server>
+    <id>apache.releases.https</id>
+    <username>your-apache-id</username>
+    <password>your-apache-password</password>
+  </server>
+</servers>
+```
+
+### Release Fluo
+
+Before starting the release process below, the following tasks should be complete:
+
+ * Create release notes for project website using GitHub issues.
+ * Perform testing and document results
+ * Start a gpg-agent to cache your gpg key to avoid entering your passphrase multiple times.
+
+   ```shell
+   gpg-connect-agent reloadagent /bye
```

Comment:
I don't think this is the correct command to start the gpg-agent. The previously documented command actually starts a gpg-agent. This may not be necessary if you're running inside a desktop environment, like GNOME or KDE, which provides a gpg-agent for you (gnome-keyring-daemon provides this in GNOME, for example).

What this gpg-connect-agent command does is clears the currently running gpg-agent's cache. The create-release-candidate script does this to help guarantee that the timer for your cached passphrase is reset before a potentially long-running build. The user still must ensure that their gpg-agent is configured with a sufficiently long timeout so that the cached passphrase won't expire during the build.